src/main/java/uk/gov/hmcts/ccd/domain/service/createcase/DefaultCreateCaseOperation.java

```diff
@@ -44,7 +45,7 @@
     private final ValidateCaseFieldsOperation validateCaseFieldsOperation;
 
     @Inject
-    public DefaultCreateCaseOperation(@Qualifier(CachedUserRepository.QUALIFIER) final UserRepository userRepository,
+    public DefaultCreateCaseOperation(@Qualifier(DefaultUserRepository.QUALIFIER) final UserRepository userRepository,
```

Comment:
DefaultCreateCaseOperation does not retrieves userRoles, so does not need the cached repository